src/app/containers/MediaPlayer/helpers/embedUrl.js

```diff
@@ -8,10 +8,10 @@ const buildBaseUrl = origin => {
   return `https://www.test.bbc${tld}`;
 };
 
-const embedUrl = ({ origin, assetId, vpid, isAmp = false }) => {
-  const avRoute = 'ws/av-embeds/articles';
+const embedUrl = ({ origin, type, assetId, isAmp = false }) => {
```

Comment:
Could we rename `assetId` to something like `requestUrl` now that `assetId` and `vpid` are combined?